### DISCUSSION TITLE
Scroll Speed in osu!mania editor

### DISCUSSION THREAD

**Discussion by theredpro-sj** [ppy/osu#30669] [Ideas]
Whenever you change the playback rate to anything other than 100% the scroll speed (this isn't caused by sv's) gets adjusted with it. This in turn can mess up player muscle memory when testing at lower playback rates as the notes come at a different scroll speed. It can become a more serious issue when testing higher star rating maps due to higher note density.

**Comment by ghost** [comment]
I second this. Adjusting the scroll speed to the playback rate is unnecessary.

**Comment by peppy** [comment]
Can you explain what you mean by this? Right now the editor slows down as I would expect it to when adjusting the playback rate:


https://github.com/user-attachments/assets/52df56d5-a934-410b-a2c9-03500f1a569c


Are you talking about a different scenario, or is this somehow not what you want?

**Reply by ghost** [reply]
I personally don't see a problem about the editor slowing down, but the problem is that when you press the "Test" button, the scroll speed is slowed down just like In the editor. Some people including me may want to practice certain parts in a slow speed(I'm not talking about the scroll speed) since the pattern is so dense or the speed is too fast etc... but when the scroll speed becomes slow, it's hard to practice. I think some people may disagree with this but I think most people would want to practice maps with a normal scroll speed.

**Reply by peppy** [reply]
The editor isn't really made for practice play, we will introduce a proper practice mode in the future.

**Reply by ghost** [reply]
> The editor isn't really made for practice play, we will introduce a proper practice mode in the future.

Ok. Thanks for the reply
